imgparse/getters.py

```diff
@@ -65,18 +65,9 @@ def read_exif_header_from_s3(image_path):
 
     bucket, key = image_path[5:].split("/", 1)
     obj = boto3.resource("s3").Object(bucket, key)
-    # Read the first 12 bytes of the image to parse the header size
-    exif_start = obj.get(Range="bytes=0-11")["Body"].read().hex()
-    # Check the header matches the format expected
-    # See here for more info: https://www.media.mit.edu/pia/Research/deepview/exif.html
-    if exif_start[:8] == "ffd8ffe1" and exif_start[12:] == "457869660000":
-        size = int(exif_start[8:12], 16)
-    else:
-        logger.warning("S3 file doesn't match expected exif header format")
-        size = 65536
 
     # Read the entire exif header for the image and return it to be parsed by exifread
-    return BytesIO(obj.get(Range=f"bytes=0-{size}")["Body"].read())
+    return BytesIO(obj.get(Range="bytes=0-65536")["Body"].read())
```

Comment:
Double checked with Zach and it seems that 64kb should be more than enough for all of our images including 6X.